website/docs/API.md

```diff
@@ -519,40 +559,34 @@ A `RenderHookOptions<Props>` object to modify the execution of the `callback` fu
 
 #### `initialProps`
 
-The initial values to pass as `props` to the `callback` function of `renderHook`.  The `Props` type is determined by the type passed to or inferred by the `renderHook` call.
+The initial values to pass as `props` to the `callback` function of `renderHook`. The `Props` type is determined by the type passed to or inferred by the `renderHook` call.
 
 #### `wrapper`
 
 A React component to wrap the test component in when rendering. This is usually used to add context providers from `React.createContext` for the hook to access with `useContext`.
 
-### `RenderHookResult<Result, Props>` object
+### `RenderHookResult` object
+
+```ts
+interface RenderHookResult<Result, Props> {
+  result: { current: Result };
+  rerender: (props: Props) => void;
+  unmount: () => void;
+}
+```
 
 The `renderHook` function returns an object that has the following properties:
 
 #### `result`
 
-```jsx
-{
-  current: Result
-}
-```
-
-The `current` value of the `result` will reflect the latest of whatever is returned from the `callback` passed to `renderHook`.  The `Result` type is determined by the type passed to or inferred by the `renderHook` call.
+The `current` value of the `result` will reflect the latest of whatever is returned from the `callback` passed to `renderHook`. The `Result` type is determined by the type passed to or inferred by the `renderHook` call.
 
 #### `rerender`
 
-```ts
-function rerender(newProps?: Props): void;
-```
-
-A function to rerender the test component, causing any hooks to be recalculated. If `newProps` are passed, they will replace the `callback` function's `initialProps` for subsequent rerenders.  The `Props` type is determined by the type passed to or inferred by the `renderHook` call.
+A function to rerender the test component, causing any hooks to be recalculated. If `newProps` are passed, they will replace the `callback` function's `initialProps` for subsequent rerenders. The `Props` type is determined by the type passed to or inferred by the `renderHook` call.
 
 #### `unmount`
 
-```ts
```

Comment:
Why remove the API here while keeping it for `fireEvent.scroll: (element: ReactTestInstance, ...data: Array<any>) => void` for instance?

Reply:
This `rerender` refers to `RenderHookResult` type which is kinda its own section. So I've put the whole type definition just above sections for first of it (`result`). Do you think we should repeat it here?

Reply:
No you're right, it makes sense!